### DISCUSSION TITLE
One drone gps visualization

### DISCUSSION THREAD

**Discussion by JonathanVogt** [ctu-mrs/mrs_uav_system#105] [Q&A] [answered]
Hello, I wanted to know, on the one drone gps simulation, what does the small frames and blue arrows depict ?

![Screenshot from 2021-08-17 16-30-49](https://user-images.githubusercontent.com/83419926/129902893-3687a113-54bb-444d-9b4f-b613defc357d.png)

Thanks,
Jonathan

**Comment by klaxalk** [comment, marked as answer]
Hello, the small frame represents the current pose of the drone, as being estimated by the UAV state estimator. The arrows are a "debugging" product of the MpcTracker. Namely, there is an internal MPC reference (the straight "line" of arrows) and the actual predicted horizon (the curved one).
